### ISSUE TITLE
Unable to paste in text - invalid format (IE).

### ISSUE THREAD

**Issue by andschdk** [olahol/react-tagsinput#84]
_Related: #35_ 

Hi,
I'm having trouble pasting content into the component in IE 11.

I'm using the component to enter multiple emails (type in or copy/paste), using the `pasteSplit` prop to separate the pasted text into multiple tags.

I've narrowed down the problem to 
```javascript
 let data = e.clipboardData.getData('text/plain')
```
(https://github.com/olahol/react-tagsinput/blob/master/src/index.js#L190)

The format `text/plain` seems to be invalid in IE 11. If I set it to ` let data = e.clipboardData.getData('Text')` instead everything works just fine in IE 11 (then it doesn't work in decent browsers).

I don't believe that common polyfills or shims like `'babel-polyfill',
    'es5-shim',
    'es6-shim'` takes care of this issue.

Can some of the contributors/owners verify this behaviour or do you know what fixes this?

I'm setting these props:
```javascript
<TagsInput
    ref='tagsInput'
    value={emails}
    addKeys={[9, 13, 32, 186, 188]} // tab, enter, space, semicolon, comma
    onlyUnique={true}
    addOnPaste={true}
    validationRegex={EMAIL_VALIDATION_REGEX}
    pasteSplit={ data => {
        return data
        .replace(/[\r\n,;]/g, ' ')
        .split(' ')
        .map(d => d.trim())
    }}
    renderTag={ props => {
        let { tag, key, onRemove, ...other } = props
        return (
            <span key={key} {...other}>
              {tag}
              <a onClick={() => onRemove(key)}>×</a>
            </span>
        )
    }}
    renderInput={ props => {
        let { onChange, value, ...other } = props
        return (
            <input
                {...other}
                type='text'
                onChange={onChange}
                value={value}
                placeholder='Add emails'
                />
        )
    }}
    renderLayout={ (tags, input) => {
        return (
            <div>
              {tags}
              {input}
            </div>
        )
    }}
    onChange={ tags => {
        this.setState({emails: tags})
    }} />
```

**Comment by andschdk** [comment]
Worth adding an inspection snapshot from IE 11 and Chrome for comparison.

**IE:**
![screen shot 2016-06-29 at 14 02 33](https://cloud.githubusercontent.com/assets/1392414/16451282/615649c2-3e02-11e6-9c7a-198ea6088914.png)

**Chrome:**
![screen shot 2016-06-29 at 14 03 05](https://cloud.githubusercontent.com/assets/1392414/16451290/69768db0-3e02-11e6-9839-386234f4a133.png)

**Comment by olahol** [comment]
I will check this out after work, I had no idea that IE used a different clipboardData interface :neutral_face:

**Comment by andschdk** [comment]
@olahol thank you - welcome to the hell of IE...

I've tried some temporary fix:
```javascript
// Check if IE
        if (e.clipboardData.getData('Text')) {
          data = e.clipboardData.getData('Text');
          var tags = pasteSplit(data);
          this._addTags(tags);
          return
        }

        // Other decent browsers
        if (e.clipboardData.getData('text/plain')) {
          data = e.clipboardData.getData('text/plain');
          var tags = pasteSplit(data);
          this._addTags(tags);
          return
        }
```
https://github.com/joblicate/react-tagsinput/commit/a40e215b1145dfe03e377e4c19ce25cdc03eedc7

Not pretty...

Tested it in IE 11, Firefox 44, Chrome 51, Safari 9.1

More info about `getData` for IE
https://msdn.microsoft.com/en-us/library/ms536436(v=vs.85).aspx

I believe that `clipboardData` is also on `window` in IE 😨

**Comment by olahol** [comment]
This is fixed in version `3.9.1`.

**Comment by andschdk** [comment]
Great work 👍 🎉
